static/bidder-info/triplelift_native.yaml

```diff
@@ -1,3 +1,4 @@
+endpoint: "https://tlx.3lift.com/s2sn/auction?supplier_id=20"
 disabled: true
```

Comment:
Just to confirm: do you want to enable this adapter as well?

Reply:
No. We just want to make the endpoint available should any publisher/PBS host want to leverage it. This is not our main adapter so it is seldom used. We're just looking to save an email asking us what the endpoint should be should anyone want to turn this on.